.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - docker build -t thelounge:main .
```

Comment:
maybe this should be in the "script"  or "install" part, building it is part of the test

Reply:
True! I'm not very familiar with the different stages Travis provides and when one should use them, but that makes sense.